2016-08-13 14:57:26 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.front.goods.GoodsAction/getGoodsList-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\GoodsMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.GoodsMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
2016-08-13 14:57:26 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.front.goods.GoodsAction/getGoodsList-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\GoodsMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.GoodsMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
2016-08-13 14:57:26 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.front.goods.GoodsAction/getGoodsList-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\GoodsMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.GoodsMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
2016-08-13 15:01:19 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.front.goods.GoodsAction/getGoodsList-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An attempt by a client to checkout a Connection has timed out.
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\GoodsMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.GoodsMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An attempt by a client to checkout a Connection has timed out.
2016-08-13 15:01:51 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-LH_ERROR_加载数据字典列表出现异常_-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An SQLException was provoked by the following failure: java.lang.InterruptedException
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\DictMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.DictMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An SQLException was provoked by the following failure: java.lang.InterruptedException
2016-08-13 15:01:52 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.front.goods.GoodsAction/getGoodsList-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An SQLException was provoked by the following failure: java.lang.InterruptedException
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\GoodsMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.GoodsMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An SQLException was provoked by the following failure: java.lang.InterruptedException
2016-08-13 15:01:52 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-LH_ERROR_加载数据字典列表出现异常_-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An SQLException was provoked by the following failure: java.lang.InterruptedException
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\DictMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.DictMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An SQLException was provoked by the following failure: java.lang.InterruptedException
2016-08-23 14:23:29 [ERROR] - R -Result.java(284) -LH_ERROR-PAGE-LH_ERROR-User-PAGE-/userHome-加载用户中心出现异常-null
2016-08-23 14:47:09 [ERROR] - org.springframework.web.servlet.DispatcherServlet -FrameworkServlet.java(497) -Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping#0': Invocation of init method failed; nested exception is java.lang.IllegalStateException: Ambiguous mapping found. Cannot map 'userFundAction' bean method 
public static com.alibaba.fastjson.JSONObject com.lhfeiyu.action.front.user.UserFundAction.getPayPasswordVerifycode(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpSession,java.lang.String)
to {[/getPayPasswordVerifycode],methods=[POST],params=[],headers=[],consumes=[],produces=[],custom=[]}: There is already 'userAction' bean method
public static com.alibaba.fastjson.JSONObject com.lhfeiyu.action.front.user.UserAction.getPayPasswordVerifycode(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpSession,java.lang.String) mapped.
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1566)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:539)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:762)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757)
	at org.springframework.context.support.AbstractApplicationContext.__refresh(AbstractApplicationContext.java:480)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java)
	at org.springframework.web.servlet.FrameworkServlet.configureAndRefreshWebApplicationContext(FrameworkServlet.java:663)
	at org.springframework.web.servlet.FrameworkServlet.createWebApplicationContext(FrameworkServlet.java:629)
	at org.springframework.web.servlet.FrameworkServlet.createWebApplicationContext(FrameworkServlet.java:677)
	at org.springframework.web.servlet.FrameworkServlet.initWebApplicationContext(FrameworkServlet.java:548)
	at org.springframework.web.servlet.FrameworkServlet.initServletBean(FrameworkServlet.java:489)
	at org.springframework.web.servlet.HttpServletBean.init(HttpServletBean.java:136)
	at javax.servlet.GenericServlet.init(GenericServlet.java:244)
	at org.eclipse.jetty.servlet.ServletHolder.initServlet(ServletHolder.java:532)
	at org.eclipse.jetty.servlet.ServletHolder.doStart(ServletHolder.java:344)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.eclipse.jetty.servlet.ServletHandler.initialize(ServletHandler.java:791)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:265)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1242)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:717)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:494)
	at org.mortbay.jetty.plugin.JettyWebAppContext.doStart(JettyWebAppContext.java:298)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.eclipse.jetty.server.handler.HandlerCollection.doStart(HandlerCollection.java:229)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.doStart(ContextHandlerCollection.java:172)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.eclipse.jetty.server.handler.HandlerCollection.doStart(HandlerCollection.java:229)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.eclipse.jetty.server.handler.HandlerWrapper.doStart(HandlerWrapper.java:95)
	at org.eclipse.jetty.server.Server.doStart(Server.java:282)
	at org.mortbay.jetty.plugin.JettyServer.doStart(JettyServer.java:65)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.mortbay.jetty.plugin.AbstractJettyMojo.startJetty(AbstractJettyMojo.java:520)
	at org.mortbay.jetty.plugin.AbstractJettyMojo.execute(AbstractJettyMojo.java:365)
	at org.mortbay.jetty.plugin.JettyRunMojo.execute(JettyRunMojo.java:523)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: java.lang.IllegalStateException: Ambiguous mapping found. Cannot map 'userFundAction' bean method 
public static com.alibaba.fastjson.JSONObject com.lhfeiyu.action.front.user.UserFundAction.getPayPasswordVerifycode(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpSession,java.lang.String)
to {[/getPayPasswordVerifycode],methods=[POST],params=[],headers=[],consumes=[],produces=[],custom=[]}: There is already 'userAction' bean method
public static com.alibaba.fastjson.JSONObject com.lhfeiyu.action.front.user.UserAction.getPayPasswordVerifycode(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpSession,java.lang.String) mapped.
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.registerHandlerMethod(AbstractHandlerMethodMapping.java:215)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.detectHandlerMethods(AbstractHandlerMethodMapping.java:187)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.initHandlerMethods(AbstractHandlerMethodMapping.java:147)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.afterPropertiesSet(AbstractHandlerMethodMapping.java:126)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping.afterPropertiesSet(RequestMappingHandlerMapping.java:126)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1625)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1562)
	... 59 more
2016-08-23 14:50:58 [ERROR] - org.springframework.web.servlet.DispatcherServlet -FrameworkServlet.java(497) -Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping#0': Invocation of init method failed; nested exception is java.lang.IllegalStateException: Ambiguous mapping found. Cannot map 'userFundAction' bean method 
public static com.alibaba.fastjson.JSONObject com.lhfeiyu.action.front.user.UserFundAction.getPayPasswordVerifycode(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpSession,java.lang.String)
to {[/getPayPasswordVerifycode],methods=[POST],params=[],headers=[],consumes=[],produces=[],custom=[]}: There is already 'userAction' bean method
public static com.alibaba.fastjson.JSONObject com.lhfeiyu.action.front.user.UserAction.getPayPasswordVerifycode(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpSession,java.lang.String) mapped.
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1566)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:539)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:762)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757)
	at org.springframework.context.support.AbstractApplicationContext.__refresh(AbstractApplicationContext.java:480)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java)
	at org.springframework.web.servlet.FrameworkServlet.configureAndRefreshWebApplicationContext(FrameworkServlet.java:663)
	at org.springframework.web.servlet.FrameworkServlet.createWebApplicationContext(FrameworkServlet.java:629)
	at org.springframework.web.servlet.FrameworkServlet.createWebApplicationContext(FrameworkServlet.java:677)
	at org.springframework.web.servlet.FrameworkServlet.initWebApplicationContext(FrameworkServlet.java:548)
	at org.springframework.web.servlet.FrameworkServlet.initServletBean(FrameworkServlet.java:489)
	at org.springframework.web.servlet.HttpServletBean.init(HttpServletBean.java:136)
	at javax.servlet.GenericServlet.init(GenericServlet.java:244)
	at org.eclipse.jetty.servlet.ServletHolder.initServlet(ServletHolder.java:532)
	at org.eclipse.jetty.servlet.ServletHolder.doStart(ServletHolder.java:344)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.eclipse.jetty.servlet.ServletHandler.initialize(ServletHandler.java:791)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:265)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1242)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:717)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:494)
	at org.mortbay.jetty.plugin.JettyWebAppContext.doStart(JettyWebAppContext.java:298)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.eclipse.jetty.server.handler.HandlerCollection.doStart(HandlerCollection.java:229)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.doStart(ContextHandlerCollection.java:172)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.eclipse.jetty.server.handler.HandlerCollection.doStart(HandlerCollection.java:229)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.eclipse.jetty.server.handler.HandlerWrapper.doStart(HandlerWrapper.java:95)
	at org.eclipse.jetty.server.Server.doStart(Server.java:282)
	at org.mortbay.jetty.plugin.JettyServer.doStart(JettyServer.java:65)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:64)
	at org.mortbay.jetty.plugin.AbstractJettyMojo.startJetty(AbstractJettyMojo.java:520)
	at org.mortbay.jetty.plugin.AbstractJettyMojo.execute(AbstractJettyMojo.java:365)
	at org.mortbay.jetty.plugin.JettyRunMojo.execute(JettyRunMojo.java:523)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
	at sun.reflect.GeneratedMethodAccessor5.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: java.lang.IllegalStateException: Ambiguous mapping found. Cannot map 'userFundAction' bean method 
public static com.alibaba.fastjson.JSONObject com.lhfeiyu.action.front.user.UserFundAction.getPayPasswordVerifycode(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpSession,java.lang.String)
to {[/getPayPasswordVerifycode],methods=[POST],params=[],headers=[],consumes=[],produces=[],custom=[]}: There is already 'userAction' bean method
public static com.alibaba.fastjson.JSONObject com.lhfeiyu.action.front.user.UserAction.getPayPasswordVerifycode(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpSession,java.lang.String) mapped.
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.registerHandlerMethod(AbstractHandlerMethodMapping.java:215)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.detectHandlerMethods(AbstractHandlerMethodMapping.java:187)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.initHandlerMethods(AbstractHandlerMethodMapping.java:147)
	at org.springframework.web.servlet.handler.AbstractHandlerMethodMapping.afterPropertiesSet(AbstractHandlerMethodMapping.java:126)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping.afterPropertiesSet(RequestMappingHandlerMapping.java:126)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1625)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1562)
	... 59 more
2016-08-23 15:15:35 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.back.shop.BackShopAction/back/getShopList-
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\ShopMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.ShopMapper.selectListByCondition-Inline
### The error occurred while setting parameters
### SQL: SELECT        A.id AS id,A.serial AS serial,A.main_status AS mainStatus,A.logic_status AS logicStatus,A.type_id AS typeId,A.type_code AS typeCode,A.deleted_at AS deletedAt,A.deleted_by AS deletedBy,A.group_id AS groupId,A.grade_id AS gradeId,A.dict_code AS dictCode,A.user_id AS userId,A.name AS name,A.password AS password,A.last_login_time AS lastLoginTime,A.last_login_ip AS lastLoginIp,A.logo AS logo,A.province AS province,A.city AS city,A.address AS address,A.licence AS licence,A.licence_paths AS licencePaths,A.organization_code AS organizationCode,A.member_num AS memberNum,A.phone AS phone,A.phone2 AS phone2,A.is_warrant AS isWarrant,A.is_seven_return AS isSevenReturn,A.is_free_postage AS isFreePostage,A.credit_margin AS creditMargin,A.grade AS grade,A.score AS score,A.is_pc_upload_allowed AS isPcUploadAllowed,A.file_paths AS filePaths,A.file_paths2 AS filePaths2,A.link_type AS linkType,A.link_id AS linkId,A.owner_id AS ownerId,A.owner_ids AS ownerIds,A.attr_str AS attrStr,A.attr_str2 AS attrStr2,A.attr_int AS attrInt,A.attr_int2 AS attrInt2,A.sequence AS sequence,A.remark AS remark,A.created_at AS createdAt,A.created_by AS createdBy,A.updated_at AS updatedAt,A.updated_by AS updatedBy      ,(       SELECT count(*) FROM goods g WHERE A.id = g.shop_id AND g.deleted_at IS NULL AND (g.main_status IS NULL OR g.main_status = 1 OR g.main_status = 76)      ) AS goodsNum      ,u.username AS userName,u.is_real_auth AS isRealAuth,u.phone AS phone      ,u.serial AS userSerial      ,pca.area_name AS cityName,pca1.area_name AS provinceName      FROM  shop  A        LEFT JOIN user u on A.user_id = u.id      LEFT JOIN user_relation ur ON ur.user_id = A.user_id AND ur.relation_type = 64             LEFT JOIN province_city_area pca ON pca.id = A.city      LEFT JOIN province_city_area pca1 ON pca1.id = A.province      WHERE 1=1         AND A.deleted_at IS NULL                                                                                                                                   ORDER BY A.id    DESC                   LIMIT 0,6
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
2016-08-23 15:16:06 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.back.shop.BackShopAction/back/getShopList-
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\ShopMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.ShopMapper.selectListByCondition-Inline
### The error occurred while setting parameters
### SQL: SELECT        A.id AS id,A.serial AS serial,A.main_status AS mainStatus,A.logic_status AS logicStatus,A.type_id AS typeId,A.type_code AS typeCode,A.deleted_at AS deletedAt,A.deleted_by AS deletedBy,A.group_id AS groupId,A.grade_id AS gradeId,A.dict_code AS dictCode,A.user_id AS userId,A.name AS name,A.password AS password,A.last_login_time AS lastLoginTime,A.last_login_ip AS lastLoginIp,A.logo AS logo,A.province AS province,A.city AS city,A.address AS address,A.licence AS licence,A.licence_paths AS licencePaths,A.organization_code AS organizationCode,A.member_num AS memberNum,A.phone AS phone,A.phone2 AS phone2,A.is_warrant AS isWarrant,A.is_seven_return AS isSevenReturn,A.is_free_postage AS isFreePostage,A.credit_margin AS creditMargin,A.grade AS grade,A.score AS score,A.is_pc_upload_allowed AS isPcUploadAllowed,A.file_paths AS filePaths,A.file_paths2 AS filePaths2,A.link_type AS linkType,A.link_id AS linkId,A.owner_id AS ownerId,A.owner_ids AS ownerIds,A.attr_str AS attrStr,A.attr_str2 AS attrStr2,A.attr_int AS attrInt,A.attr_int2 AS attrInt2,A.sequence AS sequence,A.remark AS remark,A.created_at AS createdAt,A.created_by AS createdBy,A.updated_at AS updatedAt,A.updated_by AS updatedBy      ,(       SELECT count(*) FROM goods g WHERE A.id = g.shop_id AND g.deleted_at IS NULL AND (g.main_status IS NULL OR g.main_status = 1 OR g.main_status = 76)      ) AS goodsNum      ,u.username AS userName,u.is_real_auth AS isRealAuth,u.phone AS phone      ,u.serial AS userSerial      ,pca.area_name AS cityName,pca1.area_name AS provinceName      FROM  shop  A        LEFT JOIN user u on A.user_id = u.id      LEFT JOIN user_relation ur ON ur.user_id = A.user_id AND ur.relation_type = 64             LEFT JOIN province_city_area pca ON pca.id = A.city      LEFT JOIN province_city_area pca1 ON pca1.id = A.province      WHERE 1=1         AND A.deleted_at IS NULL                                                                                                                                   ORDER BY A.id    DESC                   LIMIT 0,6
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
2016-08-23 15:16:09 [ERROR] - R -Result.java(319) -LH_ERROR-Hospital-AJAX-/back/getShopArray-加载圈子数组列表出现异常-
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\ShopMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.ShopMapper.selectListByCondition-Inline
### The error occurred while setting parameters
### SQL: SELECT        A.id AS id,A.serial AS serial,A.main_status AS mainStatus,A.logic_status AS logicStatus,A.type_id AS typeId,A.type_code AS typeCode,A.deleted_at AS deletedAt,A.deleted_by AS deletedBy,A.group_id AS groupId,A.grade_id AS gradeId,A.dict_code AS dictCode,A.user_id AS userId,A.name AS name,A.password AS password,A.last_login_time AS lastLoginTime,A.last_login_ip AS lastLoginIp,A.logo AS logo,A.province AS province,A.city AS city,A.address AS address,A.licence AS licence,A.licence_paths AS licencePaths,A.organization_code AS organizationCode,A.member_num AS memberNum,A.phone AS phone,A.phone2 AS phone2,A.is_warrant AS isWarrant,A.is_seven_return AS isSevenReturn,A.is_free_postage AS isFreePostage,A.credit_margin AS creditMargin,A.grade AS grade,A.score AS score,A.is_pc_upload_allowed AS isPcUploadAllowed,A.file_paths AS filePaths,A.file_paths2 AS filePaths2,A.link_type AS linkType,A.link_id AS linkId,A.owner_id AS ownerId,A.owner_ids AS ownerIds,A.attr_str AS attrStr,A.attr_str2 AS attrStr2,A.attr_int AS attrInt,A.attr_int2 AS attrInt2,A.sequence AS sequence,A.remark AS remark,A.created_at AS createdAt,A.created_by AS createdBy,A.updated_at AS updatedAt,A.updated_by AS updatedBy      ,(       SELECT count(*) FROM goods g WHERE A.id = g.shop_id AND g.deleted_at IS NULL AND (g.main_status IS NULL OR g.main_status = 1 OR g.main_status = 76)      ) AS goodsNum      ,u.username AS userName,u.is_real_auth AS isRealAuth,u.phone AS phone      ,u.serial AS userSerial      ,pca.area_name AS cityName,pca1.area_name AS provinceName      FROM  shop  A        LEFT JOIN user u on A.user_id = u.id      LEFT JOIN user_relation ur ON ur.user_id = A.user_id AND ur.relation_type = 64             LEFT JOIN province_city_area pca ON pca.id = A.city      LEFT JOIN province_city_area pca1 ON pca1.id = A.province      WHERE 1=1         AND A.deleted_at IS NULL                                                                                                                                               ORDER BY A.id DESC
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
2016-08-23 15:16:17 [ERROR] - R -Result.java(319) -LH_ERROR-Hospital-AJAX-/back/getShopArray-加载圈子数组列表出现异常-
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\ShopMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.ShopMapper.selectListByCondition-Inline
### The error occurred while setting parameters
### SQL: SELECT        A.id AS id,A.serial AS serial,A.main_status AS mainStatus,A.logic_status AS logicStatus,A.type_id AS typeId,A.type_code AS typeCode,A.deleted_at AS deletedAt,A.deleted_by AS deletedBy,A.group_id AS groupId,A.grade_id AS gradeId,A.dict_code AS dictCode,A.user_id AS userId,A.name AS name,A.password AS password,A.last_login_time AS lastLoginTime,A.last_login_ip AS lastLoginIp,A.logo AS logo,A.province AS province,A.city AS city,A.address AS address,A.licence AS licence,A.licence_paths AS licencePaths,A.organization_code AS organizationCode,A.member_num AS memberNum,A.phone AS phone,A.phone2 AS phone2,A.is_warrant AS isWarrant,A.is_seven_return AS isSevenReturn,A.is_free_postage AS isFreePostage,A.credit_margin AS creditMargin,A.grade AS grade,A.score AS score,A.is_pc_upload_allowed AS isPcUploadAllowed,A.file_paths AS filePaths,A.file_paths2 AS filePaths2,A.link_type AS linkType,A.link_id AS linkId,A.owner_id AS ownerId,A.owner_ids AS ownerIds,A.attr_str AS attrStr,A.attr_str2 AS attrStr2,A.attr_int AS attrInt,A.attr_int2 AS attrInt2,A.sequence AS sequence,A.remark AS remark,A.created_at AS createdAt,A.created_by AS createdBy,A.updated_at AS updatedAt,A.updated_by AS updatedBy      ,(       SELECT count(*) FROM goods g WHERE A.id = g.shop_id AND g.deleted_at IS NULL AND (g.main_status IS NULL OR g.main_status = 1 OR g.main_status = 76)      ) AS goodsNum      ,u.username AS userName,u.is_real_auth AS isRealAuth,u.phone AS phone      ,u.serial AS userSerial      ,pca.area_name AS cityName,pca1.area_name AS provinceName      FROM  shop  A        LEFT JOIN user u on A.user_id = u.id      LEFT JOIN user_relation ur ON ur.user_id = A.user_id AND ur.relation_type = 64             LEFT JOIN province_city_area pca ON pca.id = A.city      LEFT JOIN province_city_area pca1 ON pca1.id = A.province      WHERE 1=1         AND A.deleted_at IS NULL                                                                                                                                               ORDER BY A.id DESC
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
2016-08-23 15:16:20 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.back.shop.BackShopAction/back/getShopList-
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\ShopMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.ShopMapper.selectListByCondition-Inline
### The error occurred while setting parameters
### SQL: SELECT        A.id AS id,A.serial AS serial,A.main_status AS mainStatus,A.logic_status AS logicStatus,A.type_id AS typeId,A.type_code AS typeCode,A.deleted_at AS deletedAt,A.deleted_by AS deletedBy,A.group_id AS groupId,A.grade_id AS gradeId,A.dict_code AS dictCode,A.user_id AS userId,A.name AS name,A.password AS password,A.last_login_time AS lastLoginTime,A.last_login_ip AS lastLoginIp,A.logo AS logo,A.province AS province,A.city AS city,A.address AS address,A.licence AS licence,A.licence_paths AS licencePaths,A.organization_code AS organizationCode,A.member_num AS memberNum,A.phone AS phone,A.phone2 AS phone2,A.is_warrant AS isWarrant,A.is_seven_return AS isSevenReturn,A.is_free_postage AS isFreePostage,A.credit_margin AS creditMargin,A.grade AS grade,A.score AS score,A.is_pc_upload_allowed AS isPcUploadAllowed,A.file_paths AS filePaths,A.file_paths2 AS filePaths2,A.link_type AS linkType,A.link_id AS linkId,A.owner_id AS ownerId,A.owner_ids AS ownerIds,A.attr_str AS attrStr,A.attr_str2 AS attrStr2,A.attr_int AS attrInt,A.attr_int2 AS attrInt2,A.sequence AS sequence,A.remark AS remark,A.created_at AS createdAt,A.created_by AS createdBy,A.updated_at AS updatedAt,A.updated_by AS updatedBy      ,(       SELECT count(*) FROM goods g WHERE A.id = g.shop_id AND g.deleted_at IS NULL AND (g.main_status IS NULL OR g.main_status = 1 OR g.main_status = 76)      ) AS goodsNum      ,u.username AS userName,u.is_real_auth AS isRealAuth,u.phone AS phone      ,u.serial AS userSerial      ,pca.area_name AS cityName,pca1.area_name AS provinceName      FROM  shop  A        LEFT JOIN user u on A.user_id = u.id      LEFT JOIN user_relation ur ON ur.user_id = A.user_id AND ur.relation_type = 64             LEFT JOIN province_city_area pca ON pca.id = A.city      LEFT JOIN province_city_area pca1 ON pca1.id = A.province      WHERE 1=1         AND A.deleted_at IS NULL                                                                                                                                   ORDER BY A.id    DESC                   LIMIT 0,6
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
2016-08-23 15:16:24 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.back.shop.BackShopAction/back/getShopList-
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\ShopMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.ShopMapper.selectListByCondition-Inline
### The error occurred while setting parameters
### SQL: SELECT        A.id AS id,A.serial AS serial,A.main_status AS mainStatus,A.logic_status AS logicStatus,A.type_id AS typeId,A.type_code AS typeCode,A.deleted_at AS deletedAt,A.deleted_by AS deletedBy,A.group_id AS groupId,A.grade_id AS gradeId,A.dict_code AS dictCode,A.user_id AS userId,A.name AS name,A.password AS password,A.last_login_time AS lastLoginTime,A.last_login_ip AS lastLoginIp,A.logo AS logo,A.province AS province,A.city AS city,A.address AS address,A.licence AS licence,A.licence_paths AS licencePaths,A.organization_code AS organizationCode,A.member_num AS memberNum,A.phone AS phone,A.phone2 AS phone2,A.is_warrant AS isWarrant,A.is_seven_return AS isSevenReturn,A.is_free_postage AS isFreePostage,A.credit_margin AS creditMargin,A.grade AS grade,A.score AS score,A.is_pc_upload_allowed AS isPcUploadAllowed,A.file_paths AS filePaths,A.file_paths2 AS filePaths2,A.link_type AS linkType,A.link_id AS linkId,A.owner_id AS ownerId,A.owner_ids AS ownerIds,A.attr_str AS attrStr,A.attr_str2 AS attrStr2,A.attr_int AS attrInt,A.attr_int2 AS attrInt2,A.sequence AS sequence,A.remark AS remark,A.created_at AS createdAt,A.created_by AS createdBy,A.updated_at AS updatedAt,A.updated_by AS updatedBy      ,(       SELECT count(*) FROM goods g WHERE A.id = g.shop_id AND g.deleted_at IS NULL AND (g.main_status IS NULL OR g.main_status = 1 OR g.main_status = 76)      ) AS goodsNum      ,u.username AS userName,u.is_real_auth AS isRealAuth,u.phone AS phone      ,u.serial AS userSerial      ,pca.area_name AS cityName,pca1.area_name AS provinceName      FROM  shop  A        LEFT JOIN user u on A.user_id = u.id      LEFT JOIN user_relation ur ON ur.user_id = A.user_id AND ur.relation_type = 64             LEFT JOIN province_city_area pca ON pca.id = A.city      LEFT JOIN province_city_area pca1 ON pca1.id = A.province      WHERE 1=1         AND A.deleted_at IS NULL                                                                                                                                   ORDER BY A.id    DESC                   LIMIT 0,6
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'A.last_login_time' in 'field list'
2016-08-24 14:20:29 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.front.shop.ShopAction/saveOrCanelCoordinate-java.lang.String cannot be cast to java.lang.Integer
2016-08-24 15:58:47 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-LH_ERROR_加载数据字典列表出现异常_-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An attempt by a client to checkout a Connection has timed out.
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\DictMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.DictMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An attempt by a client to checkout a Connection has timed out.
2016-08-24 15:58:47 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.front.goods.GoodsAction/getGoodsList-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\GoodsMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.GoodsMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
2016-08-24 15:58:47 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-LH_ERROR_加载数据字典列表出现异常_-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\DictMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.DictMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
2016-08-24 15:58:47 [ERROR] - R -Result.java(302) -LH_ERROR-AJAX-com.lhfeiyu.action.front.goods.GoodsAction/getGoodsList-nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An attempt by a client to checkout a Connection has timed out.
### The error may exist in file [E:\lhfeiyu\works\Project_Bespeak\src\main\resources\com\lhfeiyu\dao\GoodsMapper.xml]
### The error may involve com.lhfeiyu.dao.domain.GoodsMapper.selectListByCondition
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: An attempt by a client to checkout a Connection has timed out.
